src/closedwindows/closedcard-directive.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.closedcard')
+        .directive('closedcard', [function() {
```

Comment:
Since you started this I changed all the `function()` over to `() =>`, so would be good to change this one too!